README.md

```diff
@@ -9,7 +9,12 @@ Meteor-electron strives to be the easiest way to create a desktop Meteor applica
 
 `meteor add quark:electron`
 
-After adding the package, an electron app wrapper will be while you develop.
+After adding the package, an electron app wrapper will be built and opened.
+
+This app, as well as the ready-to-distribute binaries (see [Deploy](#deploy)), is built within
+`YOUR_PROJECT_DIRECTORY/.meteor-electron`. This allows the apps to be easily located as well as
+(forthcoming) the apps to be cached for speedier startup. You should add this directory to your
+`.gitignore`.
```

Comment:
The problem isn't that the temp directory might be deleted so much as that it's unpredictable. That means that the builds can't be found after the fact, e.g. by external scripts that publish the builds.

I do think it's reasonable to let the user configure where the build directory is located. So then the only question is, what's the default? My feeling was that a known directory was better because basically everyone is going to want the directory to be known, so that they can grab the builds—and maybe some people will not want to use `.meteor-electron`, but a bunch of people will be cool with it.